packages/odyssey-react/src/components/Field/Field.module.scss

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
```

Comment:
now that we've added these new shrared components and styles - can we remove any duplicated styles from the CSS modules of Checkbox/TextInput/TextArea etc?

Reply:
Yup! During our last sync on this, we were asked to not remove those styles until after this approach was reviewed and approved.

If we feel comfortable, I am more than happy to excise all of that dupe CSS now.